.github/actions/e2e-quantic/action.yml

```diff
@@ -0,0 +1,33 @@
+name: "Quantic E2E tests"
+description: "Runs Cypress tests for Quantic"
+runs:
+  using: composite
+  steps:
+    - run: |
+        ./node_modules/.bin/sfdx force:auth:jwt:grant \
```

Comment:
I could use `npx` for all those commands.

Reply:
Can you check which version of sfdx got used iif you use `npx` tho?

IIRC, sometimes, @coveo/service-integration wants to use a specific version of the SFDX CLI simply because, well, the latest is brokey.

So I think they'd appreciate that their `package.json` is well respected ;)

Reply:
`npx` looks for the package in the `node_modules/.bin` folder, so it should be equivalent. I will add `--no-install` to every `npx` call to be sure nothing ever gets installed, but when I tested `npx` with a package that isn't installed, it prompted me.